login.go

```diff
@@ -144,17 +143,19 @@ func v2login(email, password, secondFactor string) (string, error) {
 	req := apiRequestBase("")
 	req.Method = "POST"
 
-	queryPassword := "&password=" + url.QueryEscape(password)
-	req.Uri = req.Uri + "/login?username=" + url.QueryEscape(email) + queryPassword
+	req.Uri = req.Uri + "/login"
+	req.ContentType = "application/x-www-form-urlencoded"
+	req.ShowDebug = false
+
+	data := url.Values{}
+	data.Set("username", email)
+	data.Set("password", password)
+	req.Body = data.Encode()
```

Comment:
Never mind...CLI is using a custom http client: https://github.com/franela/goreq#sending-payloads-in-the-body